templates/services/merge/validator/teku.tmpl

```diff
@@ -1,27 +1,10 @@
 {{/* teku.tmpl */}}
 {{ define "validator" }}
-  validator-import:
-    container_name: validator-import-client
-    build:
-      context: github.com/NethermindEth/teku-init-validator
-    volumes:
-      - ${VL_DATA_DIR}:/data
-      - ${KEYSTORE_DIR}:/keystore{{if .LoggingDriver}}{{if .CustomConsensusConfigs}}{{if .CustomNetworkConfigPath}}
-      - {{.CustomNetworkConfigPath}}:/network_config/config.yml{{end}}{{end}}
-    logging:
-      driver: "{{.LoggingDriver}}"
-      options:
-        max-size: "10m"
-        max-file: "10"{{end}}
-
-  validator:{{if and (ge .UID 0) (ge .GID 0)}}
-    user: "{{.UID}}:{{.GID}}"{{end}}
```

Comment:
It looks like something was lost during the merge with `develop` because this file didn't conflict during the merge, it was merged automatically. I will add it again

Reply:
Fixed [here](https://github.com/NethermindEth/sedge/pull/208/commits/4420942f8a5280d6b5fa767aa272d9fa673840ad)